cmake/SDL_ttf_Config.cmake.in

```diff
@@ -0,0 +1,15 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+
+if(NOT TARGET freetype::freetype)
+    include(${CMAKE_CURRENT_LIST_DIR}/freetypeConfigMacro.cmake)
+endif()
+
+if(NOT TARGET SDL2::SDL2)
+    set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_LIST_DIR}/")
+    include(${CMAKE_CURRENT_LIST_DIR}/SDL2ConfigMacro.cmake)
+endif()
+
+include("${CMAKE_CURRENT_LIST_DIR}/@TARGETS_EXPORT_NAME@.cmake")
+check_required_components("@PROJECT_NAME@")
```

Comment:
https://0.readthedocs.io/en/latest/git.html#check-there-is-a-new-line-at-end-of-file